.stylelintrc.js

```diff
@@ -1,3 +1,64 @@
 module.exports = {
   extends: '@wagtail/stylelint-config-wagtail',
+  rules: {
+    'scss/at-rule-no-unknown': [
+      true,
+      {
+        ignoreAtRules: [
+          'tailwind',
+          'apply',
+          'variants',
+          'responsive',
+          'screen',
+          'layer',
+        ],
+      },
+    ],
+    'no-invalid-position-at-import-rule': [
+      true,
+      {
+        ignoreAtRules: ['tailwind', 'use'],
+      },
+    ],
+    // Would be valuable for strict BEM components but is too hard to enforce with legacy code.
+    'no-descending-specificity': null,
```

Comment:
Note: As we're retooling, we could enforce it now?

Reply:
Sounds good, i'll remove it